tests/end2end/features/prompts.feature

```diff
@@ -263,6 +272,17 @@ Feature: Prompts
         And I run :prompt-accept no
         Then the javascript message "notification permission denied" should be logged
 
+    @qtwebengine_notifications
+    Scenario: notifications with ask -> false and save
+        Given I have a fresh instance
```

Comment:
I don't know how much you care about speed with these things (or how slow starting a fresh instance is) but would it be faster to just set the setting back to default afterwards?

Reply:
That'd be great, but unfortunately QtWebEngine somehow remembers those permissions (until the next restart) as well.